src/backend/utils/cache/relcache.c

```
@@ -897,6 +901,54 @@ RelationBuildRuleLock(Relation relation)
+			Anum_pg_class_relowner,
+			BTEqualStrategyNumber, F_OIDEQ,
+			ObjectIdGetDatum(relation->rd_rel->relowner));
+
+	class_rel = table_open(RelationRelationId, AccessShareLock);
+	scan = table_beginscan_catalog(class_rel, keycount, key);
+	while ((tuple = heap_getnext(scan, ForwardScanDirection)) != NULL)
+	{
+		seqoid = ((Form_pg_class) GETSTRUCT(tuple))->oid;
+	}
+
+	relation->rd_rowdSeqid = seqoid;
```

Cppcheck (2.10-2)

[warning] Uninitialized variable

(uninitvar)